### PR TITLE
[ATMOSPHERE-374] Enable Nova automatic host discovery at a set interval

### PR DESCRIPTION
**Pull request by yaguangtang**

### CODE REVIEW

**Comment by yaguangtang** [comment]
recheck

**Comment by yaguangtang** [comment]
recheck

File: roles/nova/vars/main.yml [hunk 1]
```diff
@@ -107,6 +107,7 @@ _nova_helm_values:
         ovsdb_connection: unix:/run/openvswitch/db.sock
       scheduler:
         workers: 8
+        discover_hosts_in_cells_interval: 300
```
Comment (mnaser) [inline roles/nova/vars/main.yml:110]: Let's switch this to 30, it makes more sense in our environments and it will appear faster.

**Review by mnaser — CHANGES_REQUESTED** [review]

File: roles/nova/vars/main.yml (hunk 1 above)
Comment (yaguangtang) [reply roles/nova/vars/main.yml:110]: OK

**Review by mnaser — APPROVED** [review]

**Comment by mnaser** [comment]
recheck

**Comment by mnaser** [comment]
regate

**Comment by mnaser** [comment]
regate

**Pull request merged**